src/main/java/com/laptrinhoop/Checkstyle.java

```diff
@@ -0,0 +1,22 @@
+package com.laptrinhoop.checkstyle;
+
+public class Checkstyle {
+    public void check() {
+        int a = 1;
+        int b = 2;
+        int c = a + b;
+            System.out.println(c);
+    }
+
+    public void check2() {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class 'Checkstyle' looks like designed for extension (can be subclassed), but the method 'check2' does not have javadoc that explains how to do that safely. If class is not designed for extension consider making the class 'Checkstyle' final or making the method 'check2' static/final/abstract/empty, or adding allowed annotation for the method.
<!-- __reviewdog__:ChBkZWUyZTQ0MDRkYTIyMmM0EgpjaGVja3N0eWxl -->